crypto/bn/asm/mips.pl

```diff
@@ -801,7 +801,7 @@
 #if 0
 /*
  * The bn_div_3_words entry point is re-used for constant-time interface.
- * Implementation is retained as hystorical reference.
+ * Implementation is retained as historical reference.
```

Comment:
Could also be 'hysterical', right?  ;-)

Reply:
There was something like "_corrent option_" somewhere that I didn't fix, because of the ambiguity between _correct_ and _current_. I can't find the exact sentence anymore.